container-engine-lib/lib/backend_interface/objects/logs_aggregator/logs_aggregator.go

```diff
@@ -0,0 +1,28 @@
+package logs_aggregator
+
+import (
+	"github.com/kurtosis-tech/kurtosis/container-engine-lib/lib/backend_interface/objects/container_status"
+	"net"
+)
+
+type LogsAggregator struct {
+	status container_status.ContainerStatus
+
+	// This information will be nil if the logs aggregator container isn't running
```

Comment:
Hehe, I think right now this is a lie. See my comment above

Reply:
yes i lied : )